selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/CreateFactoryFromUiWithKeepDirTest.java

```diff
@@ -147,9 +147,15 @@ private void createFactoryAndSwitchToWs() throws IOException {
       fail("Known issue https://github.com/eclipse/che/issues/7253");
     }
 
-    projectExplorer.expandPathInProjectExplorerAndOpenFile(
-        PROJECT_NAME + "/" + KEEPED_DIR + "/src" + "/main" + "/java/hello",
-        "GreetingController.java");
+    try {
+      projectExplorer.expandPathInProjectExplorerAndOpenFile(
+          PROJECT_NAME + "/" + KEEPED_DIR + "/src" + "/main" + "/java/hello",
+          "GreetingController.java");
+    } catch (TimeoutException ex) {
+      // remove try-catch block after issue has been resolved
+      fail("Known issue https://github.com/eclipse/che/issues/7959");
```

Comment:
It's better to pass "_ex_" variable into the **fail()** method to have additional info about the error in the test log.

Reply:
Please keep the output of the log short. I think a message is enough. No need to put giant stack trace to output.

Reply:
okay